### DISCUSSION TITLE
How to configure NodeJs agent

### DISCUSSION THREAD

**Discussion by surya-muthuraj0** [apache/skywalking#9731] [General]
Iam not able to configure nodeJS agent and I am facing issues in starting nodeJs agent. I've read the docs still I am not able to do it.

**Comment by kezhenxu94** [comment]
> Iam not able to configure nodeJS agent and I am facing issues in starting nodeJs agent. I've read the docs still I am not able to do it.
> 
> 

Tell us what/how you tried, what you got. Just saying it doesn't work makes no sense because we have had several releases and we have tests that proves it works.

**Reply by surya-muthuraj0** [reply]
Yeah that's the point. I don't know how to start the agent and configure it. I actually did java agent it's working but I not able to do th nodeJS agent.Sir, actually Iam new to skywalking and I have no Idea about nodeJs Agent implementation please guide us.

**Reply by kezhenxu94** [reply]
> Yeah that's the point. I don't know how to start the agent and configure it. I actually did java agent it's working but I not able to do th nodeJS agent.Sir, actually Iam new to skywalking and I have no Idea about nodeJs Agent implementation please guide us.

If you have read [the doc](https://github.com/apache/skywalking-nodejs#install-skywalking-nodejs-package-from-npmjs), you should know how to start even if you can't make it for the first attempt

**Reply by surya-muthuraj0** [reply]
I'm not able to understand the doc I've read the docs completely still I'm not able to do it.

**Reply by wu-sheng** [reply]
If you can't, you may not have nodejs developing experience. Try to ask your team who has this experience. 
This doc is already super easy.
